redhat-cop/infra-ansible
# Fixing bastion playbook based on role move

### What does this PR do?
Updating the bastion playbook to use `config-packages` instead of the removed `config-tools` role. Also eliminating the "old" inventory as all inventories should exists in the top-level `inventory` directory.

### How should this be tested?
Use the `provision-bastion/bastion.yml` playbook to stand up a new bastion host, or use the `provision-bastion/install.yml` in the even that an existing instance should be used as bastion.

### Is there a relevant Issue open for this?
N/A

### Other Relevant info, PRs, etc.
PR #152 introduced breakage due to a renamed role. 

### People to notify
cc: @redhat-cop/infra-ansible
